### PR TITLE
AWS Migration Changes

### PR DESCRIPTION
**Pull request by keckelt**
cherry-picked commit from develop

* Updates the URL of the Elastic Container Registry (ECR) repository, i.e., where the images get stored
* Updates the Cluster name from _caleydo_  to the new cluster _JKU_ASG_Cluster_

Note: For the build to succeed, the AWS Environment variables need to be updated on CircleCi.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,14 +52,14 @@ jobs:
             tasksExists=$(aws --output text ecs list-task-definitions --family-prefix ${awsFamily})
             echo "existsTaskDefinition? --${tasksExists}--"
             if [ ! -z "$tasksExists" ]; then
-              isTaskRunning=$(aws --output text ecs list-tasks --cluster caleydo --family ${awsFamily})
+              isTaskRunning=$(aws --output text ecs list-tasks --cluster JKU_ASG_Cluster --family ${awsFamily})
```
Comment (thinkh) [inline .circleci/config.yml:55]: I think, the production cluster name is `ordino-public`. 

```suggestion
              isTaskRunning=$(aws --output text ecs list-tasks --cluster ordino-public --family ${awsFamily})
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -52,14 +52,14 @@ jobs:
...
               echo "isTaskRunning? --${isTaskRunning}--"
               if [ ! -z "$isTaskRunning" ]; then
                 taskId=$(echo "${isTaskRunning}" | cut -f2)
                 echo "taskId--${taskId}--"
-                aws --output text ecs stop-task --cluster caleydo --task ${taskId}
+                aws --output text ecs stop-task --cluster JKU_ASG_Cluster --task ${taskId}
```
Comment (thinkh) [inline .circleci/config.yml:60]:
```suggestion
                aws --output text ecs stop-task --cluster ordino-public --task ${taskId}
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -52,14 +52,14 @@ jobs:
...
               fi
-              aws --output text ecs run-task --cluster caleydo --task-definition ${awsFamily} --started-by CircleCIAutoUpdate
+              aws --output text ecs run-task --cluster JKU_ASG_Cluster --task-definition ${awsFamily} --started-by CircleCIAutoUpdate
```
Comment (thinkh) [inline .circleci/config.yml:62]:
```suggestion
              aws --output text ecs run-task --cluster ordino-public --task-definition ${awsFamily} --started-by CircleCIAutoUpdate
```

**Review by thinkh — CHANGES_REQUESTED** [review]

**Comment by keckelt** [comment]
Thanks for the changes @thinkh! I've included your changes.

**Comment by thinkh** [comment]
The changes are already integrated in the main branch.

**Pull request closed without merging**